### DISCUSSION TITLE
Interface navigation accessibility

### DISCUSSION THREAD

**Discussion by rayriffy** [ppy/osu#13759] [Ideas] [answered]
Current state of lazer is very impressive, much more stable and better sound feedback. But I'm feeling that it's lacking some navigation improvements.

Introduction
---

At current builds `2021.703.0`, usually when navigate into different pages (i.e. Beatmap Listing > Song) this can be achieve by left-click a card to transition into new page. Now to navigate back in other hand, you have to either left-click on out bound area or hit `ESC` key which in both of them have a bit of the problem.

- Left-clicking outbound area has to be learned from random trial. (and it takes some to move cursor to outbound area)
- Hitting `ESC` button also takes time to move *other* hand into the farthest area of the keyboard while your hand already always positioned in mouse/trackpad

Suggestion
---

My suggestion is to implement right-clicking for navigation to go back by one level (but some area may not be suitable i.e. choosing song area), because at the moment right-click does not do anything anyways so it might be a quality of life improvements to saving some time for navigating to elsewhere faster

| ![CleanShot 2021-07-04 at 03 28 07](https://user-images.githubusercontent.com/22584594/124363914-2e3f8380-dc79-11eb-8ce9-c3a2293f2036.png) | ![CleanShot 2021-07-04 at 03 28 02](https://user-images.githubusercontent.com/22584594/124363916-313a7400-dc79-11eb-9ae7-a00ee138b0b0.png) |
| - | - |
| **Whoops, I want to go back to listing again** | **Right-click away to go back** |

**Comment by peppy** [comment, marked as answer]
There's already an issue open tracking this I believe. Right click is not the answer though.

**Reply by frenzibyte** [reply]
Yep, https://github.com/ppy/osu/issues/5063.
